data/human/culture conversations.txt

```diff
@@ -120,6 +149,38 @@ mission "Paradise Cleaner Incident"
 
 
 
+mission "Forest Fire"
+	minor
+	source
+		government "Free Worlds"
+		attributes "frontier"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`The smell of burning smoke permeates the air as you step off your craft, causing you to rush towards the nearest shelter, coughing along the way. Once there, you find people shouting, rushing about to get tape plastered onto the windows. No one seems to have noticed your arrival and none give an explanation.`
+			choice
+				`	"Hey! What's happening? I can help!" (Shout.)`
+					goto help
+				`	(Look for someone not in motion.)`
+			`	You seek out a pregnant lady sitting on a chair, a mask worn as she explains: "A large fire has sprung up nearby, and the wind is fanning it towards the spaceport. Might have been some fool playing with fire, or some pirate sending a message to a local. We don't know, could be anything around here really. We're hunkering down and hoping it'll blow over us."`
+			choice
+				`	"I have a ship out there. Let me help."`
+			`	Her eyes widen and she looks around, sharply catching the attention of one of the many people hurrying about.`
+				goto fight
+			label help
+			`	A young man who looks suspiciously like he might've been air traffic control twenty minutes ago glances at you and skids to a stop. "Oh, you're a pilot! Can your ship help?"`
+			choice
+				`	"Yes."`
+			label fight
+			`	The dock-hands scramble to get the firefighting equipment onto your ship after your proclamation, instructing you to drop the retardants onto the perimeter of the fire. You barely manage to remember what every button does before the Air Traffic Controller clears you to depart, navigating your craft towards the fire.`
```

Comment:
```suggestion
			`	The dock-hands scramble to get the firefighting equipment onto your ship after your proclamation, instructing you to drop the retardants onto the perimeter of the fire. You barely manage to remember what every button does before the air traffic controller clears you to depart. You navigate your craft towards the fire.`
```

Reply:
Can you elaborate on why you don't like the suggested change?

Reply:
Air Traffic Controller is a title and hence capitalised. It's also not good practice to cut the sentence short and add yet another short sentence at the back when you can join them together.

Reply:
Would this be better?
```suggestion
			`	The dock-hands scramble to get the firefighting equipment onto your ship after your proclamation, instructing you to drop the retardants onto the perimeter of the fire. You barely manage to remember what every button does before the Air Traffic Controller clears you to depart, and you navigate your craft towards the fire.`
```

Reply:
To me it now it feels clumsily constructed.

Reply:
You can resolve this then if you like.